### PR TITLE
Contacts section milestone

### PR DESCRIPTION
**Pull request by mutugi-c**
In this milestone, the following things were achieved:
- addition of a contact form to the Portfolio project.
- Completion of the project's mobile version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -308,6 +308,42 @@ <h2>
...
+        <div class="contacts">
+          <div class="contacts_head">
+            <h2 class="contacts_heading">Contact me</h2>
+            <p class="contacts_text">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (Ibizugbe) [inline index.html:321]: It is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) that you collect data by using the [Formspree service](https://formspree.io/html)😉. kindly set up an account and put the right link there😎.


FORMSPRE GITHUB

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @mutugi-c 👋👋, 

tryagain


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are passing👍👍
- [x] All input fields are wrapped in `form` element👍.
- [ ] Github flow has been used👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mutugi-c) [reply index.html]: Hello @Ibizugbe. I hadn't paid attention to that, thanks. Kindly try it again and leave me a message 😎.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @mutugi-c 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [ ] Good job making most of the changes requested by the previous reviewer. 🙏
 - [ ] Good job overall following the template design 💯

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -318,23 +318,23 @@ <h2 class="contacts_heading">Contact me</h2>
...
               maxlength="30"
               required
             />
             <input
               class="form-input"
-              type="email"
+              name="email"
               placeholder="bobbyfirmino@gmail.com"
               required
             />
```
Comment (TadesseDev) [inline index.html:334]: - [ ] Great job making the changes @mutugi-c, But while doing so I notice you have removed the `type` attributes from your form elements, and as part of this milestone it's among the main requirement that `all input fields have the correct TYPPE attribute`. This has a lot of advantages when it comes to accessibility.  Please make sure the name has a `text` and the email has an `email` type.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @mutugi-c 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [ ] Good job making most of the changes requested by the previous reviewer. 🙏
 - [ ] Good job overall following the template design 💯

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -93,8 +91,6 @@ Example: To build for production with parcel, use the folowing commands:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Author1**
```
Comment (amon-cofie) [inline README.md:94]: [OPTIONAL] Very well done on the readme. It looks very well built but I think that you can make this section look even more professional by replacing ```Author1``` here with your name.

**Review by amon-cofie — APPROVED** [review]
Hi @mutugi-c , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 No linter errors
🚀 Great styling 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (mutugi-c) [reply README.md:94]: Thanks, @amon-cofie 
I'll surely do that.

**Pull request merged**